DNN Platform/DotNetNuke.Abstractions/Logging/IEventLogService.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+namespace DotNetNuke.Abstractions.Logging
+{
+    using System;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// The event log service provides APIs for
+    /// managing existing Event Logs in the database.
+    /// </summary>
+    public interface IEventLogService : IEventLogger
+    {
+        /// <summary>
+        /// Clears the Event Log.
+        /// </summary>
+        void ClearLog();
+
+        /// <summary>
+        /// Deletes a specific Event Log.
+        /// </summary>
+        /// <param name="logInfo">the log info.</param>
+        void DeleteLog(ILogInfo logInfo);
+
+        /// <summary>
+        /// Get the Event Log.
+        /// </summary>
+        /// <param name="portalID">The portal id.</param>
+        /// <param name="logType">The log type.</param>
+        /// <param name="pageSize">The page size.</param>
+        /// <param name="pageIndex">The page index.</param>
+        /// <param name="totalRecords">The total records.</param>
+        /// <returns>The logs.</returns>
+        IEnumerable<ILogInfo> GetLogs(int portalID, string logType, int pageSize, int pageIndex, ref int totalRecords);
+
+        /// <summary>
+        /// Retrieves a single event log via the Log Guid.
+        /// </summary>
+        /// <param name="logGuid">A string reprenstation of the log Guid.</param>
+        /// <returns>The <see cref="ILogInfo"/>.</returns>
+        ILogInfo GetLog(string logGuid);
```

Comment:
@bdukes @mitchelsellers I add this new API following the standards that are already in place from the `DBLoggingProvider`. I am not sure of the history on this one but it appears that searching by a varchar guid representation vs the primary key of `LogEventId` will be slower. I originally wanted to add 2 methods
- GetLog(string logGuid)
- GetLog(int logEventId)

I noticed the underlying technique to read the event log is using a stored procedure. When I saw this, I decided adding the other method isn't really a good idea right now. 

I wanted to document this and share my findings